server/pulp/server/content/sources/model.py

```diff
@@ -139,7 +139,10 @@ def load_all(conf_d=None):
             if not os.path.isfile(path):
                 continue
             cfg = ConfigParser()
-            cfg.read(path)
+            list_of_read_files = cfg.read(path)
+            if not list_of_read_files:
+                log.warn("Could not read content source config file at path %s", path)
```

Comment:
I think I've observed *warn* not being included in the log while debugging satellite cases.